js/index.js

```diff
@@ -1248,10 +1248,6 @@ function DataManager() {
 				"title" : "DE Groups"
 			},
 			{
-				data : 'user.displayName',
```

Comment:
@tokland maybe I am missing something... but why did you remove the created by field?